apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -420,12 +420,81 @@ type RolloutStrategy struct {
 	// +optional
 	RollingUpdate *RollingUpdateConfig `json:"rollingUpdate,omitempty"`
 
-	// ApplyStrategy describes how to resolve the conflict if the resource to be placed already exists in the target cluster
-	// and is owned by other appliers.
+	// ConflictResolutionStrategy describes how to resolve conflicts if the resource to be placed already exists in the target cluster.
+	// +optional
+	ConflictResolutionStrategy *ConflictResolutionStrategy `json:"conflictResolutionStrategy,omitempty"`
```

Comment:
seems that ConflictResolutionStrategy and ApplyStrategy have some duplicate functionalities.

They're all about how to apply when the resource exists in the target cluster.

how about combining this new field with applyStrategy? treat them as new applyStrategy type?

Reply:
I actually found it hard to explain if they are combined. For example, they both have a type.  I think the main reason is exactly because previously the "applyStrategy" also touches some conflict resolution. Unfortunately, we can't change that anymore.